config/initializers/locations.rb

```diff
@@ -1 +1 @@
-Locations.path = Rails.root.join('lib', 'locations', 'locations.json')
+Locations.path = 'http://0.0.0.0:8080/locations.json'
```

Comment:
It still works when it's a path on the local filesystem, which is what happens [with the acceptance features](https://github.com/guidance-guarantee-programme/pension_guidance/blob/28db163d16ed04da16ac0e1b3b9fff945dc6180e/features/support/env.rb#L6).